pkg/gadgettracermanager/gadgettracermanager.go

```diff
@@ -298,9 +300,20 @@ func NewServer(conf *Conf) (*GadgetTracerManager, error) {
 		return nil, err
 	}
 
+	// Dirty hack
+	op := operators.GetRaw("KubeTempManager")
```

Comment:
That's not necessary, as the type assertions happens with a check - or do you mean we should log something in that case?

Reply:
I was thinking that it could crash if `op` was `nil`. But this `x.(T) asserts that x is not nil and that the value stored in x is of type T.` from https://stackoverflow.com/a/38818437 clarifies it. Thanks!